package.json

```diff
@@ -51,17 +51,19 @@
     "test-proto": "yarn compile-proto && yarn build-proto-roundtrip",
     "unit-core": "yarn mocha core",
     "unit-cli": "yarn mocha --testMatch cli/**/*-test.js",
+    "unit-lantern-trace": "INTERNAL_LANTERN_USE_TRACE=1 yarn mocha core/test/computed/metrics core/test/audits",
     "unit-report": "yarn mocha --testMatch report/**/*-test.js",
     "unit-treemap": "yarn mocha --testMatch treemap/**/*-test.js",
     "unit-viewer": "yarn mocha --testMatch viewer/**/*-test.js",
     "unit-flow": "bash flow-report/test/run-flow-report-tests.sh",
-    "unit": "yarn unit-flow && yarn mocha",
+    "unit": "yarn unit-flow && yarn mocha && yarn unit-lantern-trace",
```

Comment:
I think this will still run the `lantern-trace` tests as part of the normal unit test step (in addition to the new CI step you just added). IDK maybe it would be to confusing to separate this so I won't die on this hill.